torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -1001,11 +1000,12 @@ void StaticRuntime::display_nodes(
     // previous `run()` for memory planning of subsequent runs
     if (!planner_) {
       planner_ = std::make_unique<MemoryPlanner>(
-          this,
+          this->nodes(),
+          this->graph(),
+          this->outputs(),
```

Comment:
change constructor to pass in dependencies inside static runtime rather than static runtime object